doc/config.rst

```diff
@@ -1998,7 +1998,16 @@ These options influence LaTeX output. See further :doc:`latex`.
         differently or append some content after the index. For example
         ``'\\footnotesize\\raggedright\\printindex'`` is advisable when the
         index is full of long entries.
+     ``'fvset'``
+        Customization of ``fancyvrb`` LaTeX package. Defaults to
+        ``'\\fvset{fontsize=\\small}'``, because default font (Courier) used in
+        code-blocks is wider and taller than default text font (Times).
 
+        For ``'xelatex'`` and ``'lualatex'``, defaults to
+        ``'\\fvset{fontsize=auto}'``, because the default fonts are part of
+        one unified typeface family (Latin Modern OpenType).
+
+        .. versionadded:: 1.7.3
```

Comment:
At present, the target of this PR is master branch. Is this right?
If I had to choose a target, I like master branch a bit because this is not a obvious bug.

Reply:
@tk0miya Yes this is right. I chose master branch because it is not obvious bug.

Reply:
It is only default, and user can override it already to fix it without waiting for 1.8. (long time ago it was not customizable)

Reply:
ah sorry I see what you mean, ok I will edit to 1.8, OMG!

Reply:
fixed...

Reply:
Confirmed. Go ahead :-)